iosApp/Survey/Sources/Supports/Extensions/SwiftUI/View+BackButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  View+BackButton.swift
+//  Survey
+//
+//  Created by Bliss on 26/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+extension View {
+
+    func defaultBackButton(_ presentationMode: Binding<PresentationMode>) -> some View {
+        var backButton: some View {
+            Button {
+                presentationMode.wrappedValue.dismiss()
+            } label: {
+                HStack(spacing: 0) {
+                    Image(systemName: "chevron.left")
```

Comment:
I think we should create constants for system image strings also for better management and potential later usage 🙏

Reply:
Added in https://github.com/blyscuit/kmm-swiftUI-IC/pull/56/commits/8edbbc2a81db456ef46a1258d58b109b6a24d252